sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/interval/IntervalExpressionsSuite.scala

```diff
@@ -0,0 +1,183 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.expressions.interval
```

Comment:
ditto. Let's remove the `interval` directory and move this suite to the parent directory.

Reply:
I moved the interval related expressions to the separate package to not clash to date-time expressions. Though I renamed interval expressions in https://github.com/apache/spark/pull/25981/commits/f3cf7f01e060f75611768ef80c1ebce36561aded but having `Millennium` and `Millenniums` in the same name space looks confusing, doesn't it?

Reply:
New changes look better to me.